shopgate.php

```diff
@@ -171,6 +173,8 @@ public function install()
             'header',
             'adminOrder',
             'updateOrderStatus',
+            'updateQuantity',
+            'actionUpdateQuantity',
```

Comment:
The current code is tested and we had some curious behavior. The developer of this feature told me that the hook for "actionUpdateQuantity" was called twice (before updateQuantity hook was added to the code) in version 1.6.x.x thats why we added this condition "&& !empty($param['product']) && (!($param['product'] instanceof ProductCore))" to prevent the code executed twice. (This works because the params differ between each call)

We will change that in a later point. As I said the current code is working and tested.